Neos.ContentRepositoryRegistry/Classes/Command/SubprocessProjectionCatchUpCommandController.php

```diff
@@ -22,7 +23,14 @@ class SubprocessProjectionCatchUpCommandController extends CommandController
      * @Flow\Inject
      * @var ContentRepositoryRegistry
      */
-    protected $contentRepositoryRegistry;
+    protected ContentRepositoryRegistry $contentRepositoryRegistry;
+
+    /**
+     * @Flow\Inject(name="Neos.ContentRepositoryRegistry:CacheCatchUpStates")
+     * @var VariableFrontend
+     */
+    protected VariableFrontend $catchUpStatesCache;
+
```

Comment:
I wanted to suggest not to expose this non-API cache, but I realized that it's not even used, is it?
```suggestion
```

Reply:
Ah, you already fixed that one :)